metrics/chrf/chrf.py

```diff
@@ -73,31 +73,52 @@
 Produces ChrF(++) scores for hypotheses given reference translations.
 
 Args:
-    predictions: The system stream (a sequence of segments).
-    references: A list of one or more reference streams (each a sequence of segments).
-    char_order: Character n-gram order.
-    word_order: Word n-gram order. If equals to 2, the metric is referred to as chrF++.
-    beta: Determine the importance of recall w.r.t precision.
-    lowercase: Enable case-insensitivity.
-    whitespace: If `True`, include whitespaces when extracting character n-grams.
-    eps_smoothing: If `True`, applies epsilon smoothing similar
-    to reference chrF++.py, NLTK and Moses implementations. Otherwise,
-    it takes into account effective match order similar to sacreBLEU < 2.0.0.
+    predictions (list of str): The predicted sentences.
+    references (list of list of str): The references. There should be one reference sub-list for each prediction sentence.
+    char_order (int): Character n-gram order. Defaults to `6`.
+    word_order (int): Word n-gram order. If equals to `2`, the metric is referred to as chrF++. Defaults to `0`.
+    beta (int): Determine the importance of recall w.r.t precision. Defaults to `2`.
```

Comment:
One thing to note here is that the defaults in our code actually just reference the defaults in the source repo, instead of inputting these default values. For the time being, I've written what those referenced defaults are, but I think it would either be good to do one of the following:
- change the documentation here to explain that the defaults are set in another repo
- set the defaults explicitly here
Let me know which one (if either) you think is best!

Reply:
You can just redirect to their repository or their documentation I think.

Otherwise if they update their repository, our documentation will be outdated and can be misleading.